instance/models/instance.py

```diff
@@ -134,6 +152,68 @@ def status(self):
         else:
             return active_server_set[0].status
 
+    @property
+    def event_context(self):
+        """
+        Context dictionary to include in events
+        """
+        return {'instance_id': self.pk}
+
+    @staticmethod
+    def on_post_save(sender, instance, created, **kwargs):
+        """
+        Triggered by the post_save event, with `created` a boolean indicating if the object was just
+        created in the DB
+        """
+        self = instance
+        self.update_logger()
```

Comment:
@mtyaka Except that `instance` here doesn't have the meaning that it appears to have - the `on_post_save()` args need to match https://docs.djangoproject.com/en/1.8/ref/signals/#post-save - ie `instance` refers to being an instance of the model, not an `OpenEdXInstance` or `Instance` from the current file. This gets really confusing for the `Server` model, which had https://github.com/open-craft/opencraft/blob/8a4ce6f5ac23f05ba668991c7b2885f5c5550d3c/instance/models/server.py#L148 before this PR. I thought assigning it to `self` made things more obvious, and applied the pattern here too for consistency. Do you think it's a bad idea?